openeo/extra/job_management/__init__.py

```diff
@@ -382,6 +382,17 @@ def run_loop():
         self._thread = Thread(target=run_loop)
         self._thread.start()
 
+    def is_job_thread_running(self) -> bool:
+        """
+        Check if the job manager thread is still running.
+        
+        Returns:
+            bool: True if the thread is running, False otherwise
```

Comment:
This is not the parameter documentation style we use, use `:return: ...` instead